package/cloudshell/iac/terraform/services/sandbox_messages.py

```diff
@@ -8,15 +8,17 @@ def __init__(self, api: CloudShellAPISession, sandbox_id: str, service_name: str
         self._service_name = service_name
         self._write_messages_to_sandbox = write_messages_to_sandbox
 
-    def write_message(self, message: str):
+    def write_message(self, message: str, prefix: str = "", postfix: str = ""):
         if self._write_messages_to_sandbox:
             formatted_message = self._format_message(self._service_name, message)
-            self._api.WriteMessageToReservationOutput(self._sandbox_id, formatted_message)
+            self._api.WriteMessageToReservationOutput(self._sandbox_id, f'{prefix}{formatted_message}{postfix}')
 
     def _format_message(self, service_name, message):
         if ' ' in service_name:
             service_name = '"{0}"'.format(service_name)
         return '{0} {1}'.format(service_name, message)
 
     def write_error_message(self, message: str):
-        self.write_message(f'<p style="color:red">{message}</p>')
+        prefix = f'<span style="text-transform:lowercase"><p style="display:inline"><font color="red">'
+        postfix = '</p></span>'
```

Comment:
in line 22 string formatting is not needed
in line 23 you dont close the "font" tag

Reply:
"in line 22 string formatting is not needed" -> Please explain
<span style="text-transform:lowercase"> -=> my attempt to overcome the uppercase css
<p style="display:inline"> -=> is needed cause if not a new line is created